### DISCUSSION TITLE
How to mention a user or the room when piping command output into mc with --code?

### DISCUSSION THREAD

**Discussion by MTRNord** [8go/matrix-commander#172] [Q&A]
Hi I wanted to pipe the output of aureport (auditd) into a matrix room but only if there is anything to report. But if there is something to report I would like to get an alert/ping. Is there any way to prepend that ping to the piped message?

**Comment by 8go** [comment]
If on Linux, how about the appropriate Bash command to check and/or concatenate, etc.?

Powershell on Windows?
